rust/operator-binary/src/controller.rs

```diff
@@ -551,7 +551,8 @@ fn build_server_rolegroup_daemonset(
         )
         .join(" && ")])
         .add_volume_mount(BUNDLES_VOLUME_NAME, BUNDLES_DIR)
-        .add_volume_mount(LOG_VOLUME_NAME, LOG_DIR);
+        .add_volume_mount(LOG_VOLUME_NAME, LOG_DIR)
+        .resources(merged_config.resources.to_owned().into());
```

Comment:
This is different than what the other operators are doing ;)
Fine for me anyway